### PR TITLE
ConferenceCall related updates

### PR DESCRIPTION
**Pull request by zolkis**
completing #69, fixing #91, #87

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1739,7 +1742,8 @@
           </p>
           <p>
             When <a><i>TCallControl</i></a> is notified of actual call
-            disconnection it MUST follow the next steps:
+            disconnection of the <a><code>Call</code></a> object
```
Comment (marcoscaceres) [inline index.html:1745]: Just a minor nit, you don't need to put `<a>` around `<code>` blocks, as Respec will automagically link them for you. Don't worry about fixing them, I'll clean them up after the merge :)

File: index.html [hunk 2]
```diff
@@ -2270,6 +2249,153 @@
           and 3-way call.
         </p>
       </section><!-- conference-steps -->
+      <section id="conference-eventhandlers"> <!-- conference-eventhandlers -->
+        <h3>
+          Event handlers
+        </h3>
+        <p>
+          The following are the <a>event handlers</a> MUST be supported as
```
Comment (marcoscaceres) [inline index.html:2257]: Typo: missing "that" before MUST

Comment (marcoscaceres) [inline index.html:2257]: will fix locally and send a PR.

File: index.html [hunk 3]
```diff
@@ -1876,79 +1880,54 @@
...
-              <code>state</code> is not equal to <code>"active"</code> or
-              <code>held</code>, an <code>InvalidModificationError</code> MUST
-              be thrown.
+            The <code>createConference</code> method MUST run the following
+            steps when invoked on a <code>TelephonyCall</code> object, referred
+            as <var>telCall</var> in these steps:
+            <ol id="steps-createConference">
+            <li>Set the <code>state</code> of <var>telCall</var>
+            to <code>"joining"</code>.
             </li>
```
Comment (efullea) [inline index.html:1889]: Should the held call also be set to 'joining' state?

File: index.html [hunk 4]
```diff
@@ -1876,79 +1880,54 @@
...
-                </li>
-                <li>Activate the multiparty call.
+                multiparty call <var>confCall</var>.
                 </li>
                 <li>Add the two calls participating in the multiparty call to
-                the <code>calls</code> property of the multiparty call object
-                and remove them from the <code>calls</code> array of the
+                the <code>calls</code> property of the <var>confCall</var>
+                object and remove them from the <code>calls</code> array of the
                 <a>TelephonyManager</a> object.
```
Comment (efullea) [inline index.html:1915]: As one of the calls could eventually be a ConferenceCall It should be clarified that just those of type TelephonyCall need to be added to the calls array in confcall and removed from the calls array in TelephonyManager

**Comment by efullea** [comment]
I miss a statement on setting the state of the individual calls to 'multiparty' after completion of the conf call

**Comment by zolkis** [comment]
@efullea, > "Should the held call also be set to 'joining' state?"

Do you mean that when the join starts, every call object should be updated with the state 'joining'?
This is problematic, and I don't see the value. If the request fails, the implementation would need to restore the original state that the network has set, which means maintaining old values (and track if they change from the network). This is too much complication and slow-down for no gain for the dialer app. 

IMO it is enough if the one call from where the 'createConference' method is called is set to 'joining'. The network (and telephony middleware) will update the state of the calls soon after the join is successful. For instance oFono returns a list of calls participating in the conference, and the implementation can update the state of each to 'multiparty' (yes, I will update the steps to reflect this).

**Comment by zolkis** [comment]
See PR #112

**Pull request merged**